src/io/itk-dicom/dicom.cpp

```diff
@@ -135,24 +231,43 @@ const json import(FileNamesContainer &files) {
   // Does this affect series organization?
   seriesFileNames->SetLoadPrivateTags(false);
 
-  using SeriesIdContainer = std::vector<std::string>;
+  // These are not necessarily the DICOM SeriesUIDs due to possible mangling.
+  // Use only as internal keys.
   const SeriesIdContainer &gdcmSeriesUIDs = seriesFileNames->GetSeriesUIDs();
 
-  for (auto gdcmSeriesUID : gdcmSeriesUIDs) {
-    FileNamesContainer fileNames =
-        seriesFileNames->GetFileNames(gdcmSeriesUID.c_str());
+  SeriesIndexType curSeriesIndex;
+  for (auto seriesUID : gdcmSeriesUIDs) {
```

Comment:
why renaming from `gdcmSeriesUID` to `seriesUID`? Less changes below if not doing that.

Reply:
I'll be renaming to volumeID, so there will be much bigger changes happening.